index.js

```diff
@@ -5,17 +5,30 @@ const BinWrapper = require('bin-wrapper');
 
 const version = '0.33.3'
 const base = 'https://github.com/saucelabs/saucectl/releases/download';
-const bin = new BinWrapper()
-	.src(`${base}/v${version}/saucectl_${version}_mac_32-bit.tar.gz`, 'darwin', 'x86')
-	.src(`${base}/v${version}/saucectl_${version}_mac_64-bit.tar.gz`, 'darwin', 'x64')
-	.src(`${base}/v${version}/saucectl_${version}_linux_32-bit.tar.gz`, 'linux', 'x86')
-	.src(`${base}/v${version}/saucectl_${version}_linux_64-bit.tar.gz`, 'linux', 'x64')
-	.src(`${base}/v${version}/saucectl_${version}_win_32-bit.zip`, 'win32', 'x86')
-	.src(`${base}/v${version}/saucectl_${version}_win_32-bit.zip`, 'win32', 'x64')
-	.src(`${base}/v${version}/saucectl_${version}_win_64-bit.zip`, 'win64', 'x64')
-    .dest(path.join(__dirname, 'bin'))
-    .use(process.platform.startsWith('win') ? 'saucectl.exe' : 'saucectl')
-	.version(`v${version}`);
+const binWrapper = (binInstallURL) => {
+    const bw = new BinWrapper();
+
+    if (binInstallURL) {
```

Comment:
Should we also check that url is valid (at least that this is really a url?)

Reply:
`bin-wrapper` will throw an exception if the URL is not reachable. I edited the message related to this.

Reply:
could the if statement only wrap `.src` methods as the other are the same?

Reply:
good idea.